en/docs/insights/programmatic-access-choreo-insights-api.md

```diff
@@ -0,0 +1,60 @@
+# Programmatic Access to Insights API
+
+The insights page in the Choreo Console fetches data from the Insights API exposed over the Internet. Occasionally, some users might need to fetch these data from external systems for various use cases. One such use case can be to implement their own dashboards based on these data.
+
+Choreo provides a secure and user-friendly method to achieve this requirement. The steps are outlined below.
+
+## Become a Member of the Choreo System Organization
+
+1. Sign in to [https://devportal.choreo.dev](https://devportal.choreo.dev) using your credentials.
+
+2. Visit [https://devportal.choreo.dev/choreosystem](https://devportal.choreo.dev/choreosystem) to access the developer portal of Choreo System organization.
+
+    If you are accessing the Choreo System organization’s developer portal for the first time, you will need request access to the organization by clicking on the **Request Access** button.
+
+    ![Request-Access](../assets/img/insights/request-access.png){.cInlineImage-full}
+
+3. Once access has been requested, you will be automatically added as a member of the Choreo System organization.
+
+    Verify this by inspecting the organization drop-down(Choreo System organization should be visible).
+
+    ![Choreo-System-Home](../assets/img/insights/choreo-system-home.png){.cInlineImage-full}
+
+## Create an Application & Generate Keys
```

Comment:
```suggestion
## Create an application and generate keys
```

Reply:
Have an introductory sentence before the steps.

e.g., To create an application and generate keys, follow the steps below: